src/org/lwjgl/demo/opengl/SimpleDrawElements.java

```diff
@@ -37,7 +39,7 @@ public void run() {
             glfwDestroyWindow(window);
             keyCallback.free();
             wsCallback.free();
-            if (debugProc != null)
+            if ( debugProc != null )
```

Comment:
Why change formatting?

Reply:
The code uses both with spaces and no spaces between parens, I just changed formatting to make it consistent.